File: src/S1/BOofOne/JS/studentHelper.js
// studentHelper.js

function showNameAt(students, index) {
    if (index >= 0 && index < students.length) {
        console.log(`Student: ${students[index]} at index ${index}`);
    } else {
        console.log(`Error: Index ${index} is out of bounds of total ${students.length}.`);
    }
}

module.exports = { showNameAt };

File: src/S1/BOofOne/Java/StudentHelper.java
// StudentHelper.java

public class StudentHelper {
    public static void showNameAt(String[] students, int index) {
        if (index >= 0 && index < students.length) {
            System.out.printf("Student: %s at index %d%n", students[index], index);
        } else {
            System.out.printf("Error: Index %d is out of bounds.%n", index);
        }
    }
}
